plugins/com.gwtplugins.gdt.eclipse.suite/project_templates/client_server_shared/client/src/_PACKAGENAME_/ui.gwt.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  When updating your version of GWT, you should also update this DTD reference,
+  so that your app can take advantage of the latest GWT module capabilities.
+-->
+<!DOCTYPE module PUBLIC "-//Google Inc.//DTD Google Web Toolkit 2.10.0//EN"
+  "http://gwtproject.org/doctype/2.10.0/gwt-module.dtd">
+<module rename-to='example'>
+  <!-- Inherit the core Web Toolkit stuff.                        -->
+  <inherits name='com.google.gwt.user.User'/>
+
+  <!-- Inherit the default GWT style sheet.  You can change       -->
+  <!-- the theme of your GWT application by uncommenting          -->
+  <!-- any one of the following lines.                            -->
+  <inherits name='com.google.gwt.user.theme.clean.Clean'/>
+  <!-- <inherits name='com.google.gwt.user.theme.standard.Standard'/> -->
+  <!-- <inherits name='com.google.gwt.user.theme.chrome.Chrome'/> -->
+  <!-- <inherits name='com.google.gwt.user.theme.dark.Dark'/>     -->
+
+  <!-- Other module inherits                                      -->
+  <inherits name='${PACKAGENAME}.shared'/>
+
+  <!-- Specify the app entry point class.                         -->
+  <entry-point class='${PACKAGENAME}.ui.Main'/>
+
+  <!-- Specify the paths for translatable code                    -->
+  <source path='ui'/>
+
+  <!-- allow Super Dev Mode -->
+  <add-linker name="xsiframe"/>
```

Comment:
No longer needed since GWT 2.7, almost 10 years https://github.com/gwtproject/gwt/commit/22865ad27d54e2eb03a017feefc482c59afbc7fe

Reply:
removed the xsiframe